getting-started/windows-calico/requirements.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Requirements 
+description: Review requirements for the standard install for Calico for Windows.
+canonical_url: '/getting-started/windows-calico/requirements'
+---
+
+### About {{site.prodnameWindows}}
+
+Because the Kubernetes and {{site.prodname}} control components do not run on Windows yet, a hybrid Linux/Windows cluster is required. {{site.prodnameWindows}} standard installation is distributed as a **.zip archive**. 
+
+### What's supported in this release
+
+✓ Install: Manifest install for Kubernetes 2 clusters
+
+✓ Platforms: Kubernetes, EKS
+
+✓ Networking: Calico CNI with VXLAN, or other supported CNI
+
+### Requirements
+
+#### CNI and networking options
+
+The following table summarizes the networking options and considerations.
+
+| Networking              | Components                                                   | **Value/Content**                                            |
+| ----------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| {{site.prodname}} VXLAN | Windows CNI plugin:<br/>calico.exe<br /><br />Linux: {{site.prodname}} for policy and networking | {{site.prodname}}'s VXLAN overlay, supports:<br/><br />- VXLAN overlay, which can traverse most networks.<br/>- Auto-configured node-to-node routing<br/>- {{site.prodname}} IPAM and IP aggregation (with some limitations)<br/>- Both etcd and Kubernetes API datastore drivers<br/>**Note**: VXLAN runs on UDP port 4789 (this is the only port supported by Windows), remember to open that port between your {{site.prodname}} hosts in any firewalls / security groups. |
+| Cloud provider          | Windows CNI plugin: win-bridge.exe<br /><br />Linux: {{site.prodname}} policy-only | A useful fallback, particularly if you have a Kubernetes cloud provider that automatically installs inter-host routes. {{site.prodname}} has been tested with the standard **win-bridge.exe** CNI plugin so it should work with any networking provider that ultimately uses win-bridge.exe to network the pod (such as the Azure CNI plugin and cloud provider). |
+
+#### Datastores
+
+Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Windows node/Kubernetes cluster must be the same as the datastore for the Linux control node. (You cannot mix datastores in a {{site.prodnameWindows}} implementation.)
+
+#### Kubernetes version 
+
+- Versions 1.17, 1.16, or 1.15
```

Comment:
I checked with @lwr20, we should support k8s from 1.16+

Reply:
Yes.  The intent would be to match this: 
https://docs.projectcalico.org/getting-started/kubernetes/requirements#kubernetes-requirements

Reply:
Fixed